### PR TITLE
Feature/update activities

### LINKED ISSUE #30: Make activity card arrows static

### PR DESCRIPTION
**Pull request by espressoGoddess**
### Files Changed:
- Activities.css
- activityData.js

### What Has Changed: 
- I added my activities (🎊) finally.
- I updated the activity card to be a little bigger
- The buttons now stay on the outsides without moving depending on the size of text inside
- The text is a bit smaller now

### Closes:
Closes #30

### CODE REVIEW

File: client/src/Components/Activities/Activities.css [hunk 1]
```diff
@@ -16,14 +17,14 @@ h2 {
 
 .activity-card {
   display: flex;
-  justify-content: center;
+  justify-content: space-between;
   align-items: center;
-  height: 300px;
-  width: 40%;
+  text-align: center;
```
Comment (jfogiato) [inline client/src/Components/Activities/Activities.css:22]: YES

File: client/src/sampleData/activityData.js [hunk 2]
```diff
@@ -204,6 +204,104 @@ const activityData = [
     activityId: 41,
     activity: 'Put on an irresistible song and wiggle your body the weirdest way you can. Decide on the movements based on listening to how your body wants to move, stretch, and shake out, NOT on if the dance moves are good. The more like Ace Ventura: Pet Detective meets hippie at Woodstock the better',
     feelings: [ 'bad' ]
+  }, 
+  {
+    "activityId": 42,
+    "activity": "Make up and sing a song in the weirdest voice. Play around with different volume levels and pitches and have fun!",
```
Comment (jfogiato) [inline client/src/sampleData/activityData.js:210]: Please do this and send it to the group

**Review by jfogiato — APPROVED** [review]
Looks great! TY Amber for the work here :)

File: client/src/sampleData/activityData.js (hunk 2 above)
Comment (chrissycooper) [reply client/src/sampleData/activityData.js:210]: +10000000000

File: client/src/Components/Activities/Activities.css (hunk 1 above)
Comment (KatherineBlaine) [reply client/src/Components/Activities/Activities.css:22]: Center align looks great for the activity cards!

File: client/src/sampleData/activityData.js [hunk 3]
```diff
@@ -204,6 +204,104 @@ const activityData = [
...
+    "feelings": [
+      "bad",
+    ]
+  },
+  {
+    "activityId": 1,
+    "activity": "Create a cozy blanket fort with lots of pillows, and take a nap or watch your favorite movie.",
+    "feelings": [
+      "bad",
+    ]
```
Comment (KatherineBlaine) [inline client/src/sampleData/activityData.js:304]: Love these activities!!

**Review by KatherineBlaine — APPROVED** [review]
Looks great Amber!!!

**Pull request merged**